apps/backend/app/scrap/update_csv.py

```
@@ -117,14 +118,42 @@ def main():
+    # ────────────────────────────────────────────────────────────────
+
+
+    # ──────────────── ① 구(행정구)별 평균 행 제거 ────────────────
+    data = [row for row in data if row["school_name"] != "평균"]
+
+    # ──────────────── ② % 기호 제거 후 학업성취도 내림차순 정렬 ────────────────
+    def _avg(row):
+        v = row.get("average", "").replace("%", "").strip()
+        return float(v) if v else -1
+    data.sort(key=_avg, reverse=True)
+    for r in data: r.pop("average", None)
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)